### PR TITLE
Check if Presto is not running before erasing RPM

### PR DESCRIPTION
**Pull request by kokosing**
Check if Presto is running before erasing RPM

### CODE REVIEW

File: presto-server-rpm/src/main/rpm/preremove [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Pre erase script
```
Comment (findepi) [inline presto-server-rpm/src/main/rpm/preremove]: file is called `preremove`, is that not enough?

File: presto-server-rpm/src/main/rpm/preremove [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+if ! /etc/init.d/presto status | grep -q 'Not running'; then 
+    echo "Please stop the Presto service before erasing RPM"
```
Comment (findepi) [inline presto-server-rpm/src/main/rpm/preremove]: Is it what other RPMs do? eg. postgres?

Comment (kokosing) [reply presto-server-rpm/src/main/rpm/preremove]: I do not know, I checked that once but I have found anything interesting.

**Comment by kokosing** [comment]
@nezihyigitbasi ping.

**Pull request closed without merging**